#try:
  #  print('hello')
    #int(input())
   # print(x)
#except ValueError:
   # print("mistake")
#except NameError:
   # print('mistake')
#except:
  #  print('error')
try:
    num=int(input("put number"))
    print(num+a)

except ValueError:
    print("put number!")

except:
    print("mistake ")